backend/src/main/app.ts

```diff
@@ -22,6 +22,7 @@ import ProjectRoute from './routes/ProjectRoute';
 import CallRoute from './routes/CallRoute';
 import TransactionRoute from './routes/TransactionRoute';
 import EventRoute from './routes/EventRoute';
+import BusinessRoute from './routes/BusinessRoute';
 
```

Comment:
Remove the UserRoute import from this file since in the main the file has been deleted, but the import is still here